inc/c_util/worker_thread.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef WORKER_THREAD_H
+#define WORKER_THREAD_H
+
+#include "umock_c/umock_c_prod.h"
+#ifdef __cplusplus
+extern "C" {
+#endif 
+
+    /* this is the callback that is to be called each time a batch process is scheduled */
```

Comment:
>each time [](http://example.com/codeflow?start=49&length=9)

nit: If the batch process is scheduled twice, the callback may only be called once. This comment implies a one-to-one mapping of callbacks and schedule_process calls.

Reply:
Added more explanation for this